Unosquare.CodeChallenge/WarehouseRESTfulAPI/Controllers/companies.cs

```diff
@@ -33,7 +33,7 @@ public IActionResult getCompany([FromRoute]string id)
                 if (String.IsNullOrEmpty(id)) return this.BadRequest();
```

Comment:
ASP.NET is already able to serialize to value types, no need to receive a string and cast it to int

Reply:
Im sending the id encoded base64 as security measure to avoid send the number of the item as integer.

Reply:
I'll change the variable name for idEncoded

Reply:
the change is on the commit f4f6e2d1a270fce4c21d7acf8db531a02cde94fe